server/routes/caseNotes/viewAll/caseNotesView.ts

```diff
@@ -23,13 +23,23 @@ export default class CaseNotesView {
           {
             html: `<p class="govuk-body">
                     <b>${ViewUtils.escape(row.subject)}</b>
-                    <br><br>${ViewUtils.nl2br(ViewUtils.escape(row.body))}</p>`,
+                    <br><br>${ViewUtils.nl2br(ViewUtils.escape(this.getCharacters(row.body, 250)))}
+                    <p/><a href="${row.caseNoteLink}" class="govuk-link">Read More</a>
+                    </p>`,
           },
         ]
       }),
     }
   }
 
+  private getCharacters(text: string, characterCount: number): string {
+    const truncatedCharacters = text.slice(0, characterCount)
+    if (truncatedCharacters.length === characterCount) {
+      return `${truncatedCharacters}...`
```

Comment:
I feel like it would be good to have a test for this behaviour - we don't tend to test our view classes directly, other than in End to End tests. Could it be worth extracting this to a function in its own file (or like `PresenterUtils` or something similar) so we could write a test for it?